NativeCode/AudioPluginUtil.h

```diff
@@ -493,3 +505,7 @@ void DeclareEffect(
     UnityAudioEffect_GetFloatBufferCallback getfloatbuffercallback,
     InternalEffectDefinitionRegistrationCallback registereffectdefcallback
     );
+
+} // namespace AudioPluginUtil
+
+using namespace AudioPluginUtil;
```

Comment:
:grimacing: Is this to avoid making users change their code?
If it's broken today, I'd prefer to have users update their plugins than to have this using statement in the header.

Reply:
This fix is necessary to resolve class name conflicts between player code and dlls. And partially yes, it will avoid user define classes on global namespace. Actually, "class Mutex" seems conflicts bettwen player code and dlls. In some case, it'll cause the issue about function call mismatching.
(e.g.)
When the constructor of Mutex(Mutex::Mutex) will be called into dlls, Mutex::Mutex in player code will be called. (However, it depends on the platform.)

The one of the other solutions is renaming class name Thread to HogeThread. However we should check all class names on global namespace.

Reply:
Right, I think I wasn't clear, sorry.
I understand why `Mutex` got moved into the new namespace.
The part I don't like is the `using` statement in the header - I think the code that uses this should import the namespace or explicitly use `AudioPluginUtil::Mutex`

Reply:
Okay, I'll remove `using namespace AudioPluginUtil` in the header & it'll be moved on .each cpp file.

Reply:
Okay, I've added the "AudioPluginUtil::" prefix for all referenced classes/functions/constants.